subshell

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+const [path, ...command] = process.argv.slice(2);
+
+const { exec } = require('child_process');
+
+const cmd = exec(`${command.join(' ')}`, { cwd: path });
+
+cmd.stdout.on('data', data => {
+  console.log(data);
+});
```

Comment:
This should also work and be a bit terser:

```
cmd.stdout.on('data', console.log);
```

Reply:
yep you're right, point-free style it's better